app/models/channel.rb

```diff
@@ -168,7 +168,7 @@ def update_last_verification_timestamp
   def should_register_channel_for_promo
     promo_running = Rails.application.secrets[:active_promo_id].present?  # Could use PromosHelper#active_promo_id
     publisher_enabled_promo = self.publisher.promo_enabled_2018q1?
-    promo_running && publisher_enabled_promo && verified_changed? && verified
+    promo_running && publisher_enabled_promo && saved_change_to_verified? && verified
```

Comment:
Is this change related to rails upgrade?

Reply:
Yes, it's related to https://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods/Dirty.html

From what I understand, `changed?` was confusing because at-a-glance, it could be a change to the unsaved version or to the saved version.